.gitignore

```diff
@@ -1,6 +1,5 @@
 *.gem
 webhook.lock
 .ruby_version
-*.log
-logs/
+logs/*.log
```

Comment:
This should log to `/var/log/puppet-webhook` or the like, rather than in the gem directory.

Reply:
Agreed, I just haven't gotten around to it yet.